routes/indicators.py

```diff
@@ -64,9 +66,19 @@ def package_base_indicators_data(point_data_list):
             di[indicator][era] = dict()
             for mi, scenario_li in enumerate(model_li):
                 model = base_dim_encodings["model"][mi]
+                # Skip impossible combinations of era and model.
+                if era == "historical" and model != "Daymet":
+                    continue
+                elif era != "historical" and model == "Daymet":
+                    continue
```

Comment:
Good thinking! Removes unnecessary cruft from the resulting JSON and fewer loops made through the code